WooCommerce/Classes/ViewRelated/Products/Edit Product/BottomSheetListSelector/ProductTypeBottomSheetListSelectorCommand.swift

```diff
@@ -0,0 +1,102 @@
+import Yosemite
+
+private extension ProductType {
+    /// Title shown on the action sheet.
+    ///
+    var actionSheetTitle: String {
+        switch self {
+        case .simple:
+            return NSLocalizedString("Simple product",
+                                     comment: "Action sheet option Simple when the user wants to change the Product type")
```

Comment:
nit: maybe "Action sheet option when the user wants to change the Product type to simple product" (and similar for the other options)?

Reply:
👍 done here 2fe635cf92b2e62ba0f4310a39826176cc598c65